src/pages/courses/courses.js

```diff
@@ -67,154 +74,939 @@ const useStyles = makeStyles(theme => ({
+    .split(" ")
+    .slice(0, previewDescriptionWords)
+    .join(" ");
+
+  const _courseCreator = ({ client, query, id, setCourseCreator }) => {
     client
-      .request(listAllCourses, { private: true })
+      .request(query, { id })
       .then(response => {
-        setCourses({ ...courses, courses: response.listCourses });
+        setCourseCreator(response.searchUser);
       })
       .catch(error => {
-        setCourses({ courses: [], ...error.response });
+        console.log(error.response);
       });
-  }, []);
+  };
+
+  const _removeCourse = ({
+    client,
+    query,
+    id,
+    setCourseRemoved,
+    setCourses,
+    listAllCourses,
+    enqueueSnackbar
+  }) => {
+    client
+      .request(query, { id })
+      .then(response => {
+        enqueueSnackbar("Course removed", {
+          variant: "success",
+          autoHideDuration: 5000
+        });
+
+        setCourseRemoved(true);
+
+        _listAllCourses({
+          client,
+          setCourses,
+          query: listAllCourses,
+          privateCourses: false
+        });
+      })
+      .catch(error => {
+        enqueueSnackbar("Error on course remove", {
+          variant: "error",
+          autoHideDuration: 8000
+        });
+
+        console.log(error.response);
+
+        setCourseRemoved(false);
+      });
+
+    setDialogOpen(!dialogOpen);
+  };
 
   return (
-    <Box p={5}>
+    <Grid item lg={4} md={6} sm={12} xs={12} key={course.id} display="flex">
+      <Card>
+        <CardContent className={classes.cardContent}>
+          <Typography color="textSecondary" gutterBottom>
+            {course.title}
+          </Typography>
+
+          <Typography
+            color="textSecondary"
+            className={`${classes.date} ${classes.centerIconText}`}
+          >
+            <EventAvailableIcon />
+            Star {moment(course.start).format("MMMM Do YYYY h:mm a")}
+            <EventBusyIcon />
+            End {moment(course.end).format("MMMM Do YYYY h:mm a")}
+          </Typography>
+
+          <Typography
+            color="textSecondary"
+            className={`${classes.marginSvgIcon} ${classes.centerIconText}`}
+          >
+            <LockOpenIcon /> {(course.private && "Private") || "Public"}
+          </Typography>
+
+          <Typography variant="h5" component="h2">
+            {descriptionSplitted}
+          </Typography>
+        </CardContent>
+
+        <CardActions>
+          <IconButton
+            className={clsx(classes.expand, {
+              [classes.expandOpen]: expanded
+            })}
+            onClick={() => {
+              _courseCreator({
+                client,
+                query: searchCourseCreator,
+                id: course.creator,
+                setCourseCreator
+              });
+
+              handleExpandClick();
+            }}
+            aria-expanded={expanded}
+            aria-label="Show more"
+          >
+            <ExpandMoreIcon />
+          </IconButton>
```

Comment:
Add a **tooltip** here. Users can be confused with the icon.
(and you know this might happen, some of them aren't so smart) :laughing: